openedx/core/djangoapps/oauth_dispatch/tests/test_client_credentials.py

```diff
@@ -63,10 +63,11 @@ def test_jwt_access_token(self):
             'client_id': application.client_id,
             'client_secret': application.client_secret,
             'scope': ' '.join(scopes),
-            'token_type': 'jwt'
+            'token_type': 'jwt',
+            'user': self.user
```

Comment:
Why have you modified the post body? The endpoint doesn’t even recognize this field.

Reply:
i will remove this.